specification/draft/index.html

```diff
@@ -905,89 +905,95 @@ <h3>Verifiable Presentation of DSCSA Stakeholder Credentials</h3>
       <p>
         Verifiable Presentations may be used to combine and present credentials. They can be packaged in such a way that
         the authorship of the data is verifiable. OCI generates a Verifiable Presentation after [[vc-data-model]]
-        specification.
+        specification. OCI extends the <a href="https://www.w3.org/TR/vc-data-model/#jwt-decoding">JWT Encoding</a> and
+        [[RFC7519]] section 4.1 by defining its own optionality and usage guidelines.
       </p>
       <section>
         <h3>Verifiable Presentation Metadata</h3>
         <table class="simple">
           <tr>
             <th><b>Term</b></th>
             <th><b>Description</b></th>
-            <th><b>Full IRI</b></th>
           </tr>
           <tr>
-            <td><b>jti</b></td>
+            <td><b>`jti`</b></td>
             <td>
-              This field represents the id property of the verifiable presentation being generated.
+              A digital wallet MAY include a `jti` field in the JWT payload. This field represents the id property of
+              the Verifiable Presentation.
             </td>
-            <td>https://w3c.github.io/vc-data-model/#jwt-encoding</td>
           </tr>
           <tr>
-            <td><b>iss</b></td>
+            <td><b>`iss`</b></td>
             <td>
-              Represents the holder property of the verifiable presentation being generated.
+              A digital wallet SHALL include a `iss` field in the JWT payload. This field represents the holder property
+              of the Verifiable Presentation and SHALL mimic the credential subject <a>DID</a> of the embedded
+              Verifiable Presentation.
```

Comment:
> and SHALL mimic the credential subject <a>DID</a> of the embedded Verifiable Presentation.

not sure about "mimic" and confused by "of the embedded Verifiable Presentation". I understand the requirement to be: I must enter the VC subject DID in the iss field of the VP. If this is correct, why not say,

> A digital wallet SHALL include an `iss` field in the JWT payload. This field represents the holder property of the Verifiable Presentation and SHALL be the Verifiable Credential's subject <a>DID</a>.

Reply:
Yep, I messed up here. Ofc it's about the VC. I am going to "borrow" your version.